### ISSUE TITLE
Python 3.4 fails to launch with a TypeError

### ISSUE THREAD

**Issue by amjith** [dbcli/mycli#144]
```
vagrant@debian-jessie:~$ mycli -u root
Version: 1.4.0
Chat: https://gitter.im/dbcli/mycli
Mail: https://groups.google.com/forum/#!forum/mycli-users
Home: http://mycli.net
Traceback (most recent call last):
  File "/home/vagrant/py3venv/bin/mycli", line 9, in <module>
    load_entry_point('mycli==1.4.0', 'console_scripts', 'mycli')()
  File "/home/vagrant/py3venv/lib/python3.4/site-packages/click/core.py", line 664, in __call__
    return self.main(*args, **kwargs)
  File "/home/vagrant/py3venv/lib/python3.4/site-packages/click/core.py", line 644, in main
    rv = self.invoke(ctx)
  File "/home/vagrant/py3venv/lib/python3.4/site-packages/click/core.py", line 837, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/vagrant/py3venv/lib/python3.4/site-packages/click/core.py", line 464, in invoke
    return callback(*args, **kwargs)
  File "/home/vagrant/py3venv/lib/python3.4/site-packages/mycli/main.py", line 590, in cli
    mycli.run_cli()
  File "/home/vagrant/py3venv/lib/python3.4/site-packages/mycli/main.py", line 317, in run_cli
    print('Thanks to the contributor -', thanks_picker([author_file, sponsor_file]))
  File "/home/vagrant/py3venv/lib/python3.4/site-packages/mycli/main.py", line 664, in thanks_picker
    return choice([x.split('*')[1].strip() for x in contents if x.startswith('*')])
  File "/home/vagrant/py3venv/lib/python3.4/site-packages/mycli/main.py", line 664, in <listcomp>
    return choice([x.split('*')[1].strip() for x in contents if x.startswith('*')])
TypeError: startswith first arg must be bytes or a tuple of bytes, not str
```

This is because of using the `open()` function from the `codecs` module. For regular `open()` it works fine in Python 3. 

1. Use regular open() (at least just for python 3). 
2. Check if regular open() behaves correctly for unicode chars in the file.

**Comment by tsroten** [comment]
I've had success with using `io.open()`. It works well on Python 2 and 3. It supports the *encoding* parameter, which defaults to whatever `locale.getpreferredencoding()` returns. In Python 3, `io.open()` is simply an alias for the built-in `open()`. 

So, maybe instead of `from codecs import open` you could try `from io import open`.

**Comment by amjith** [comment]
Thank you for researching this. I think the io.open() will work very nicely. I'll make that change. :)

**Comment by amjith** [comment]
This is now fixed in master 3153fc9348ae6e2b38aea550970e1cbc5df7847b. Thanks @tsroten for the tip. 

It will be released in the next version.
